legend-depot-store-metrics/src/main/java/org/finos/legend/depot/store/metrics/store/api/QueryMetricsStore.java

```diff
@@ -0,0 +1,23 @@
+//  Copyright 2021 Goldman Sachs
+//
```

Comment:
do we need two sepaate interfaces... merge unto one and call ins QueryMetricsStore